packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,445 @@
+		sceneId,
+	);
+	const valueDB = this.getValueDB();
+
+	if (!valueDB.hasMetadata(levelValueId)) {
+		valueDB.setMetadata(levelValueId, {
+			...ValueMetadata.UInt8,
+			label: `Level (${sceneId})`,
+		});
+	}
+	if (!valueDB.hasMetadata(dimmingDurationValueId)) {
+		valueDB.setMetadata(dimmingDurationValueId, {
+			...ValueMetadata.Duration,
+			label: `Dimming duration (${sceneId})`,
+		});
+	}
+
+	valueDB.setValue(levelValueId, level);
+	valueDB.setValue(dimmingDurationValueId, dimmingDuration);
+
+	return true;
+}
+
+@API(CommandClasses["Scene Actuator Configuration"])
+export class SceneActuatorConfigurationCCAPI extends CCAPI {
+	public supportsCommand(
+		cmd: SceneActuatorConfigurationCommand,
+	): Maybe<boolean> {
+		switch (cmd) {
+			case SceneActuatorConfigurationCommand.Get:
+				return this.isSinglecast();
+			case SceneActuatorConfigurationCommand.Set:
+				return true;
+		}
+		return super.supportsCommand(cmd);
+	}
+
+	protected [SET_VALUE]: SetValueImplementation = async (
+		{ property, propertyKey },
+		value,
+	): Promise<void> => {
+		if (propertyKey == undefined) {
+			throwMissingPropertyKey(this.ccId, property);
+		} else if (typeof propertyKey !== "number") {
+			throwUnsupportedPropertyKey(this.ccId, property, propertyKey);
+		}
+		if (property === "level") {
+			if (typeof value !== "number") {
+				throwWrongValueType(
+					this.ccId,
+					property,
+					"number",
+					typeof value,
+				);
+			}
+
+			// We need to set the dimming duration along with the level
+			const node = this.endpoint.getNodeUnsafe()!;
+			const dimmingDuration = node.getValue<Duration>(
+				getDimmingDurationValueID(this.endpoint.index, propertyKey),
+			);
+			// If dimmingDuration is missing, we must abort
+			if (dimmingDuration == undefined) {
+				throw new ZWaveError(
+					`The "dimmingDuration" property cannot be changed before the level is known!`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			await this.set(propertyKey, dimmingDuration, value);
+		} else if (property === "dimmingDuration") {
+			if (!(value instanceof Duration)) {
+				throwWrongValueType(
+					this.ccId,
+					property,
+					"Duration",
+					typeof value,
+				);
+			}
+
+			// We need to set the leve along with the dimming duration
+			const node = this.endpoint.getNodeUnsafe()!;
+			const level = node.getValue<number>(
+				getLevelValueID(this.endpoint.index, propertyKey),
+			);
+			// If level is missing, we must abort
+			if (level == undefined) {
+				throw new ZWaveError(
+					`The "level" property cannot be changed before the dimming duration is known!`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			await this.set(propertyKey, value, level);
+		} else {
+			// setting dimmingDuration value alone not supported,
+			// to be added when setting Duration values is supported
+			throwUnsupportedProperty(this.ccId, property);
+		}
+
+		// Verify the current value after a delay
+		this.schedulePoll({ property, propertyKey });
```

Comment:
I don't think it is mission critical to verify the change. I only want to do this when it is important, e.g. lights, thermostats and config params (which might not get accepted).

Reply:
Done.